ui/component/fileRender/view.jsx

```diff
@@ -186,9 +198,16 @@ class FileRender extends React.PureComponent<Props> {
     // Return viewer
     return viewer || unsupported;
   }
-
   render() {
-    const { isText } = this.props;
+    const { isText, embedUrl } = this.props;
+
+    if (embedUrl) {
```

Comment:
Instead of returning early with almost the same markup, just use classnames.

`classnames({ "file-render": !embedUrl, ...`